hera_xblocks/end_survey/end_survey/end_survey.py

```diff
@@ -0,0 +1,124 @@
+import pkg_resources
+from web_fragments.fragment import Fragment
+from xblock.core import XBlock
+from xblock.fields import Integer, Scope, String, List
+from xblock.fields import JSONField
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+from xblockutils.resources import ResourceLoader
+from xblock.runtime import Runtime
+from django.contrib.auth.models import User
+from student.models import get_user_by_username_or_email
+
+loader = ResourceLoader(__name__)
+
+@XBlock.wants('user')
+class EndSurveyXBlock(StudioEditableXBlockMixin, XBlock):
+    """
+    XBLock response to take the surver from the student.
+    When he passed the lesson.
+    """
+
+    editable_fields = ('data',)
+    data = JSONField(default={})
```

Comment:
regroup xblock fields and xblock settings attributes

Reply:
done

Reply:
not done, separate please fields from xblock settings properties